x-pack/plugin/ccr/qa/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexSecurityIT.java

```diff
@@ -111,6 +111,53 @@ public void testFollowIndex() throws Exception {
         }
     }
 
+    public void testCreateAndFollowIndex() throws Exception {
+        final int numDocs = 16;
+        final String indexName1 = "index3";
+        final String indexName2 = "index4";
```

Comment:
See my comment above about the index names; this is especially onerous with the variable names not being lined up with the concrete index names (e.g., `indexName1` is lined up with `index3`).